file-formats/fugr/fugr.json

```diff
@@ -15,44 +17,41 @@
       "properties": {
         "description": {
           "title": "Description",
-          "description": "Description for a Function Group",
+          "description": "Description of the ABAP object",
           "type": "string",
-          "maxLength": 40
+          "maxLength": 60
         },
-        "masterLanguage": {
-          "title": "Master Language",
-          "description": "Master language of the development object\nAll translatable text shall be maintained in this language",
+        "originalLanguage": {
+          "title": "Original Language",
+          "description": "Original language of the ABAP object",
```

Comment:
"Original language of the ABAP **development** object"   sounds better,
"Original language of the ABAP object" makes me think only to classes/interfaces artifacts

Reply:
This change would imply that the description is also changed for all the other ABAP object types. I would prefer to change this in an separate pull request

Reply:
ok, leave it as it is